prefect_docker/host.py

```diff
@@ -42,14 +40,6 @@ class DockerSettings(Block):
         default=None,
         description="The maximum number of connections to save in the pool.",
     )
-    credstore_env: Dict[str, Any] = Field(
-        default_factory=dict,
-        title="Credential Store Environment Variables",
-        description=(
-            "Override environment variables when calling "
-            "the credential store process."
-        ),
-    )
```

Comment:
Is `credstore_env` a valid kwarg when instantiating a Docker client with a base URL?

Reply:
It is.